package.json

```diff
@@ -9,12 +9,13 @@
   "module": "dist/index.es.js",
   "jsnext:main": "dist/index.es.js",
   "engines": {
-    "node": ">=8",
-    "npm": ">=5"
+    "node": ">=10",
+    "npm": ">=6"
   },
   "scripts": {
     "test": "react-scripts test",
     "test:watch": "react-scripts test --env=jsdom",
+    "test:coverage": "yarn test --coverage",
```

Comment:
I get the same handlebars warnings that travisci is getting for this. The solution is to add the following to the root of your `package.json`:

```json
"resolutions": {
   "handlebars" : "4.5.0"
 },
```

Reply:
For clarification, you'll need to run `yarn install` to get the updates to your `yarn.lock` after the change is made.